notification-service/internal/handler/handler.go

```diff
@@ -0,0 +1,18 @@
+package handler
+
+import (
+	"github.com/gin-gonic/gin"
+)
+
+type Handler struct {
+}
+
+func NewHandler() *Handler {
+	return &Handler{}
+}
+
+func (h *Handler) InitRoutes() *gin.Engine {
+	router := gin.New()
+	router.GET("/metrics", h.metrics)
+	return router
+}
```

Comment:
You could implement server wrapper with `/metrics` endpoint registered by default and use it in all your services.

Reply:
Ok, thx for recommendation